dockerGrader/client/dockerGraderClient.py

```diff
@@ -0,0 +1,158 @@
+import argparse, boto3, botocore, csv, math, logging, requests, threading, time, json
+
+REQUEST_URL="http://{ip}:{port}/json/{project}/{netId}"
+RESULT_PATH="ta/grading/{project}/{netId}.json"
+METADATA_PATH="projects/{project}/users/{googleId}/curr.json"
```

Comment:
This is their full submission, not just metadata.  When I was generating meta.json from here before, I was downloading from this location, but then I was popping the payload (which contains b64 student code) to leave only metadata.